Dockerfile

```diff
@@ -1,20 +1,21 @@
 FROM postgres:9.5
 
-ENV RUST_VERSION 1.8.0
 ENV PATH ~/.cargo/bin/:$PATH
 
 RUN apt-get update \
-  && apt-get install -y --no-install-recommends ca-certificates curl git make gcc postgresql-server-dev-$PG_MAJOR=$PG_VERSION python-pip \
-  && mkdir -p /tmp/build \
-  && curl -o /tmp/build/rust-${RUST_VERSION}.tar.gz -SL https://static.rust-lang.org/dist/rust-${RUST_VERSION}-x86_64-unknown-linux-gnu.tar.gz \
-  && tar -xzf /tmp/build/rust-${RUST_VERSION}.tar.gz -C /tmp/build/ \
-  && sh /tmp/build/rust-${RUST_VERSION}-x86_64-unknown-linux-gnu/install.sh \
+  && apt-get install -y --no-install-recommends \
+  ca-certificates curl git make gcc postgresql-server-dev-$PG_MAJOR=$PG_VERSION python-pip \
```

Comment:
Wanna just friendly pull as I don't know what you want me to do?

On Sat, Jun 11, 2016 at 7:50 AM Rui Marinho <notifications@github.com>
wrote:

> In Dockerfile
> <https://github.com/posix4e/jsoncdc/pull/60#discussion_r66706835>:
>
> >  ENV PATH ~/.cargo/bin/:$PATH
> >
> >  RUN apt-get update \
> > -  && apt-get install -y --no-install-recommends ca-certificates curl git make gcc postgresql-server-dev-$PG_MAJOR=$PG_VERSION python-pip \
> > -  && mkdir -p /tmp/build \
> > -  && curl -o /tmp/build/rust-${RUST_VERSION}.tar.gz -SL https://static.rust-lang.org/dist/rust-${RUST_VERSION}-x86_64-unknown-linux-gnu.tar.gz \
> > -  && tar -xzf /tmp/build/rust-${RUST_VERSION}.tar.gz -C /tmp/build/ \
> > -  && sh /tmp/build/rust-${RUST_VERSION}-x86_64-unknown-linux-gnu/install.sh \
> > +  && apt-get install -y --no-install-recommends \
> > +  ca-certificates curl git make gcc postgresql-server-dev-$PG_MAJOR=$PG_VERSION python-pip \
>
> I was thinking about aligning && so perhaps:
>
> && apt-get install -y --no-install-recommends \
>   ca-certificates curl git make gcc postgresql-server-dev-$PG_MAJOR=$PG_VERSION python-pip
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/posix4e/jsoncdc/pull/60/files/c81a8951b57ca53c89e237e8cf0161d3842540b0#r66706835>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAxN260PIYQcp6eLcvxpmzF3bC1Bbtfdks5qKss3gaJpZM4IyoKn>
> .
>
-- 
Sent from my mobile 4045076749
Alex Newman